src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -98,8 +130,8 @@ public int getYSize() {
 
     public GuiCraftConfirm(final InventoryPlayer inventoryPlayer, final ITerminalHost te) {
         super(new ContainerCraftConfirm(inventoryPlayer, te));
-        this.xSize = 238;
-        this.ySize = displayMode.getYSize();
+        this.craftingTree = new GuiCraftingTree(this, 9, 19, 203, 192);
+        recalculateScreenSize();
```

Comment:
Maybe redundant as its already called in `initGui()`, dismiss if not relevant

Reply:
xSize/ySize are used in some calls below, like setScrollbar